modules/gitops-installing-argocd-cli-on-macos.adoc

```diff
@@ -0,0 +1,68 @@
+// Module is included in the following assemblies:
+//
+// * installing_gitops/installing-argocd-gitops-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argocd-cli-on-macos"]
+= Installing the {gitops-title} CLI on macOS
+
+For macOS, you can download the {gitops-shortname} `argocd` CLI as a `tar.gz` archive.
+
+.Procedure
+
+. Download the relevant CLI tool.
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/argocd-cli/1.12.0/argocd-cli-darwin-amd64.tar.gz[macOS on Intel]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/argocd-cli/1.12.0/argocd-cli-darwin-arm64.tar.gz[macOS on ARM]
+
++
+You can also download any version of the {gitops-shortname} `argocd` CLI by navigating to that version’s corresponding directory in the link:https://mirror.openshift.com/pub/openshift-v4/clients/argocd-cli/[`argocd` CLI client download mirror].
```

Comment:
See my previous comment on how to potentially reword this. :slightly_smiling_face: